tests/upgrades/test_client.py

```diff
@@ -232,7 +175,7 @@ class TestScenarioUpgradeNewClientAndPackageInstallation:
 
     @pytest.mark.post_upgrade
     def test_post_scenario_postclient_package_installation(
```

Comment:
```suggestion
    def test_post_scenario_post_client_package_installation(
```
I guess if we change the others we can change this one too.